synapse/lib/slabseqn.py

```diff
@@ -61,7 +61,7 @@ def nextindx(self):
             int: The next insert offset.
         '''
         indx = 0
-        with s_lmdbslab.Scan(self.lenv, self.db) as curs:
+        with s_lmdbslab.Scan(self.lenv, self.db, dupsort=False) as curs:
```

Comment:
We should only use kwarg param syntax for kwargs.  Future self ( that wants to modify args ) thanks you ;)

Reply:
ok